LibraryApplication/LibraryApplication/EditLibrary.aspx.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace LibraryApplication
+{
+    public partial class EditLibrary : System.Web.UI.Page
+    {
+        private const string LibrariesUrl = "~/Libraries.aspx";
+        private const string GetStatesQuery = @"
+            SELECT *
```

Comment:
Avoid using the `*` operator in SQL queries. Explicitly naming your columns makes it easier for reviewers to read and understand your code.